packages/pigeon/lib/dart_generator.dart

```diff
@@ -283,6 +292,7 @@ void _writeFlutterApi(
     codecName = _getCodecName(api);
     _writeCodec(indent, codecName, api, root);
   }
+  indent.addln('');
```

Comment:
You don't need to separate it out retroactively, but for future reference when you find something like these various output-format cleanups while working on a big PR, the way I would recommend handling it is pausing briefly, making a new branch to do the cleanup and send it out, then come back to the main PR separately. That way the big PR doesn't become even bigger, but you don't forget about the cleanup.

Reply:
Will do!